views.py

```diff
@@ -130,7 +142,7 @@ def get_sprint_finish_page():
     try:
         run_id = int(request.args.get('run_id', -1))
         played = request.args.get('played', False)
-        return render_with_data('play_finish.html', run_id=run_id, played=played)
+        return render_with_data('play_finish.html', type='sprint', run_id=run_id, played=played)
```

Comment:
I like having the sprint type here, but in the future we would likely want some way to figure this out on the frontend, so we can separate the frontend and backend.